projects/showcase/cypress/e2e/popup.cy.ts

```diff
@@ -1,26 +1,31 @@
-import { E2eDriver } from '../support/e2e-driver';
+import { E2eDriver } from "../support/e2e-driver";
 
-describe('Popup', () => {
-  context('Given I am on the Popup showcase', () => {
-    let driver = new E2eDriver();
+describe("Popup", () => {
+  context("Given I am on the Popup showcase", () => {
+    let { beforeAndAfter, given, when, get, then } = new E2eDriver();
+
+    beforeAndAfter();
 
     beforeEach(() => {
-      driver.visitMapPage('/demo/popup').waitForMapToIdle();
+      ({ given, when, get, then } = new E2eDriver());
+      when.visitMapPage("/demo/popup");
+      when.waitForMapToIdle();
     });
 
-    context('When the map loads', () => {
+    context("When the map loads", () => {
       it('Then I should see a popup dialog containing "Hello World !"', () => {
-        driver.assert.helloWorldPopupExists();
+        then(get.mglPopup()).shouldExist();
+        then(get.mapLibrePopUpContent()).should("have.text", "Hello world !");
```

Comment:
Doesn't the driver has a `shouldHaveText` method?

Reply:
You mean Assertable, right?
No, not yest, I'll add it.